hedera-node/src/main/java/com/hedera/services/legacy/stream/RecordStream.java

```diff
@@ -85,7 +85,7 @@
 	Platform platform;
 	MiscRunningAvgs runningAvgs;
 	boolean inFreeze;
-	String recordStreamsDirectory;
+	public final String recordStreamsDirectory;
```

Comment:
This change should be reverted.

Reply:
Just wanted to log the chosen directory in `ServicesMain.init`...added a `getRecordStreamsDirectory()` to be more kosher now. 😀